src/ext/java/org/opentripplanner/ext/flex/FlexParameters.java

```diff
@@ -0,0 +1,57 @@
+package org.opentripplanner.ext.flex;
+
+import java.time.Duration;
+
+/**
+ * Define parameters used to configure flex. For further documentation on these parameters, look
+ * at the {@link org.opentripplanner.standalone.config.sandbox.FlexConfig} class which implements
+ * this interface. The flex package does not use all parameters defined here. Some parameters are
+ * passed into the street search(AStar) as part of a flex use-case. We keep them here for
+ * completeness and simplicity (just one interface).
+ */
+public interface FlexParameters {
+  /**
+   * See {@link org.opentripplanner.standalone.config.sandbox.FlexConfig}
+   */
+  Duration maxTransferDuration();
+  /**
+   * See {@link org.opentripplanner.standalone.config.sandbox.FlexConfig}
+   */
+  Duration maxFlexTripDuration();
+  /**
+   * See {@link org.opentripplanner.standalone.config.sandbox.FlexConfig}
+   */
+  Duration maxAccessWalkDuration();
+  /**
+   * See {@link org.opentripplanner.standalone.config.sandbox.FlexConfig}
+   */
+  Duration maxEgressWalkDuration();
+
+  /**
+   * This defines the default values. This will be used by the OTP configuration and by tests,
+   * avoid using this directly.
+   */
+  static FlexParameters defaultValues() {
```

Comment:
What is the reason this is a method rather than a constant?

Reply:
Not anything big, but there are a few technical differences. A method encapsulate the instance slightly better, but in this case a new instance i created every time -- it does not matter.